frontend/js/src/artist/ArtistPage.tsx

```diff
@@ -407,20 +481,21 @@ export default function ArtistPage(): JSX.Element {
             </div>
           )}
         </div>
-        <div className="albums full-width scroll-start">
-          <h3 className="header-with-line">Albums</h3>
-          <div className="cover-art-container dragscroll">
-            {albumsByThisArtist.map(getReleaseCard)}
-          </div>
-        </div>
-        {Boolean(alsoAppearsOn?.length) && (
+        {Object.entries(groupedReleaseGroups).map(([type, rgGroup]) => (
           <div className="albums full-width scroll-start">
-            <h3 className="header-with-line">Also appears on</h3>
-            <div className="cover-art-container dragscroll">
-              {alsoAppearsOn.map(getReleaseCard)}
+            <div className="listen-header">
+              <h3 className="header-with-line">{type}</h3>
+              <SortingButtons sort={sort} setSort={setSort} />
+            </div>
+            <div
+              className={`cover-art-container dragscroll ${
+                rgGroup.length <= COVER_ART_SINGLE_ROW_COUNT ? "single-row" : ""
+              }`}
+            >
+              {rgGroup.map(getReleaseCard)}
```

Comment:
While you are in here and now that you've added a type for secondary_type on the RGs, I wonder if we should add the secondary types in getReleaseCard?
On the one hand we have the info so we could display it, on the other it seems that it would be redundant now that we have the secondary types in the subheaders.
Your call.

Reply:
I don't think we should display the secondary types on the release cards, as that'll be too much information on the card.